### PR TITLE
feat: adding item CSV import capability

### PR DESCRIPTION
**Pull request by gabrielfs7**
Allow import Item CSV

https://oat-sa.atlassian.net/browse/ADF-105

### CODE REVIEW

File: actions/class.ItemImport.php [hunk 1]
```diff
@@ -50,7 +50,10 @@ protected function getAvailableImportHandlers()
 
         foreach (array_keys($returnValue) as $key) {
             if ($returnValue[$key] instanceof \tao_models_classes_import_CsvImporter) {
-                $importer = new CsvImporter();
+                //@TODO @FIXME Inject this in the proper way
```
Comment (boajer) [inline actions/class.ItemImport.php:53]:
```suggestion
```
Could it be dropped from this comment or inject in a proper way in scope of this PR?

Comment (gabrielfs7) [reply actions/class.ItemImport.php:53]: Yes. this will be fixed. But I do not want to do it before get reviews to avoid rework :) 

Thanks

Comment (gabrielfs7) [reply actions/class.ItemImport.php]: Done

**Review by habemuscode — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by voknil — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**